S2VX.Game/Story/Note/Notes.cs

```diff
@@ -34,13 +34,13 @@ public class Notes : CompositeDrawable {
         public float NoteOutlineThickness { get; set; }
 
         // TODO: Remove these defaults and use command classes
-        public Color4 PerfectColor { get; set; } = S2VXColorConstants.LightYellow;
-        public Color4 EarlyColor { get; set; } = S2VXColorConstants.White;
-        public Color4 LateColor { get; set; } = S2VXColorConstants.DarkYellow;
-        public Color4 MissColor { get; set; } = S2VXColorConstants.Red;
-        public float PerfectThreshold { get; set; } = 30;
-        public float HitThreshold { get; set; } = 100;
-        public float MissThreshold { get; set; } = 200;
+        public static Color4 PerfectColor { get; set; } = S2VXColorConstants.LightYellow;
+        public static Color4 EarlyColor { get; set; } = S2VXColorConstants.White;
+        public static Color4 LateColor { get; set; } = S2VXColorConstants.DarkYellow;
+        public static Color4 MissColor { get; set; } = S2VXColorConstants.Red;
+        public static float PerfectThreshold { get; set; } = 30;
+        public static float HitThreshold { get; set; } = 100;
+        public static float MissThreshold { get; set; } = 200;
```

Comment:
In general, I think [making stateful member variables static is bad practice](https://stackoverflow.com/questions/7026507/why-are-static-variables-considered-evil), even if it cuts down on some references. We're probably okay right now with how we are carefully writing our tests and not setting these variables, but imagine this maybe hyperbolic scenario: Test1 sets MissThreshold to 999999 seconds, Test2 has nothing to do with testing miss threshold but now behaves/works incorrectly due to Test1's configuration.

At the very least, I would remove all the setters and treat these as constants.

Reply:
Ah yeah, I was thinking these would never be changed. Missed removing the setters.